ui/raidboss/data/02-arr/dungeon/haukke_manor.ts

```diff
@@ -78,7 +77,7 @@ const triggerSet: TriggerSet<Data> = {
       // Suppression included since 2 Handmaiden's spawn at the same time
       id: 'Haukke Normal Ladys Handmaiden',
       type: 'AddedCombatant',
-      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '424', capture: false }),
+      netRegex: { npcNameId: '424', capture: false },
```

Comment:
This will break.  `addedCombatantFull` and `addedCombatant` are not the same thing.  Maybe we need a separate PR to just make them the same thing, in the same way that `ability` and `abilityFull` are now the same thing?

Reply:
emmmm sorry I assume `addedCombatantFull ` it's like `abilityFull`, didn't really check it.

Reply:
`addedCombatantFull` and `addedCombatant` are matching same line, am I correct?

So if the trigger doesn't really use the matches it's safe to replace it?

Reply:
No, both of these use `npcNameId`, which isn't in `addedCombatant`.  Only `addedCombatantFull` has `npcNameId`.  See: https://github.com/quisquous/cactbot/blob/main/resources/netregexes.ts#L220-L247

I am suggesting that maybe this "optimization" is not worth doing anymore and we should combine the two so that they are the same, e.g.
```typescript
  /**
   * matches: https://github.com/quisquous/cactbot/blob/main/docs/LogGuide.md#line-03-0x03-addcombatant
   * @deprecated Use `addedCombatant` instead
   */
  static addedCombatantFull(params?: NetParams['AddedCombatant']): CactbotBaseRegExp<'AddedCombatant'> {
    return this.addedCombatant(params);
  }
```

Reply:
Good idea

Reply:
And I don't understand, why "both of these use `npcNameId`"? `Responses.killAdds` use no matches.

Reply:
In a seprated PR, too?

Reply:
Yeah, please.

Mostly this patch is so large that if something goes awry in either the logic patches or the conversion patch, I don't want to have to revert everything.